mantidproject/mantid
# Update GSASIIRefineFitPeaks system test input to file which gives a good fit

As of revision 3390, it appears that not having a good fit is now a throw-able offence in GSAS-II. This PR changes the input file of the system test to one which gives a good enough fit.

*Edit:* [changeset 3392](https://subversion.xray.aps.anl.gov/trac/pyGSAS/changeset/3392/trunk/GSASIIdataGUI.py) fixed this bug in GSAS-II, but it's probably still worth having a proper fit in the sytem tests

**To test:**

Install the latest version of GSASII (from Linux, you can do this with `python scripts/GSAS-II/install_gsas_proxy.py -b`), then run `./systemtest -R GSASIIRefineFitPeaks`. Make sure tests pass

Fixes #22438 

**Release Notes** 
Internal change, doesn't need to be in the release notes

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
